src/scenes/wizards_waltz.inc.c

```diff
@@ -1,37 +1,316 @@
+    GameInfo.cyclePosition += 1;
+}
+
+
+// [func_08044e60] ENGINE Func_01 - Set Tutorial Flag
+void waltz_beatscript_set_tutorial(u32 arg0) {
+    GameInfo.isTutorial = arg0;
+}
+
+
+// [func_08044e74] MAIN - Unload(?)
+void waltz_main_unload(void) { /* Stub Function */ }
+
+
+// !TODO - Currently impossible without an ASM hack. See: https://decomp.me/scratch/Kk2Ec
+// [func_08044e78] CUE Behaviour - Spawn
 #include "asm/scenes/wizards_waltz/asm_08044e78.s"
 
-#include "asm/scenes/wizards_waltz/asm_08044f94.s"
 
-#include "asm/scenes/wizards_waltz/asm_08044fc0.s"
+// [func_08044f94] CUE Behaviour - Unknown
+u32 waltz_cue_unknown(u32 arg0, u32 arg1, u32 arg2) {
+    if (arg2 > (GameInfo.cycleInterval + func_0800c3a4(0x30))) {
+        return 1; }
+    else {
+        return 0; }
+}
+
+
+// [func_08044fc0] CUE Behaviour - Despawn
+void waltz_cue_despawn(u32 arg0, u32 *arg1) {
+    func_0800fc70(arg1[0]);
+}
+
+
+// [func_08044fcc] CUE Event - Hit
+void waltz_cue_hit(u32 arg0, struct struct_080179f4_sub *arg1) {
+    u32 isTutorial;
+
+    // Play animation: "sprout_grow"
+    func_08010064(arg1->unk0, D_08932d2c, 0, 1, 0x7f, 0);
+
+    // Cycle through frames of "girl_happy" if isTutorial flag is not set.
+    isTutorial = GameInfo.isTutorial;
+    if (!isTutorial) {
+        GameInfo.girlState = 1;
+
+        // Increment frame for "girl_happy" if not at the maximum of 5.
+        if (GameInfo.flowerCount <= 5) {
+            GameInfo.flowerCount += 1;
+        }
+
+        // Play animation: "girl_happy"
+        func_08010064(GameInfo.girlEntity, D_08932e64, GameInfo.flowerCount, 0, isTutorial, isTutorial);
+    }
+}
+
+
+// [func_0804503c] CUE Event - Barely
+void waltz_cue_barely(u32 arg0, struct struct_080179f4_sub *arg1) {
+    u32 temp;
+    u32 isTutorial;
+
+    // Set unknown parameter.
+    temp = ((u32) ~func_08018054()) >> 0x1f;
+    if (arg1->unk6 > 0x200) {
+        temp ^= 1;
+    }
+
+    // Play animation: "sprout_eaten"
+    func_08010064(arg1->unk0, D_08932cbc, 0, 1, 0x7f, 0);
+    // Unknown function.
+    func_0800ff44(arg1->unk0, temp);
+
+    // Cycle through frames of "girl_sad" if isTutorial flag is not set.
+    isTutorial = GameInfo.isTutorial;
+    if (!isTutorial) {
+        GameInfo.girlState = 2;
+
+        // Decrement frame for "girl_sad" if not at the minimum of 0.
+        if (GameInfo.flowerCount) {
+            GameInfo.flowerCount -= 1;
+        }
+
+        // Play animation: "girl_sad"
+        func_08010064(GameInfo.girlEntity, D_08932ea4, GameInfo.flowerCount, 0, isTutorial, isTutorial);
+    }
+
+    // Unknown function - likely related to score.
+    func_0800bc40();
+}
+
+
+// [func_080450d0] CUE Event - Miss
+void waltz_cue_miss(u32 arg0, struct struct_080179f4_sub *arg1) {
+    // Unknown function - likely related to score.
+    func_0800bc40();
+}
+
 
-#include "asm/scenes/wizards_waltz/asm_08044fcc.s"
+// [func_080450dc] MAIN - Input Event
+void wizard_main_unknown(void) { /* Stub Function */ }

```

Comment:
So I assume this is some special stub function? If this is a parameter of some kind of global "input event" function, and it happens to be empty in Wizard's Waltz, it should probably be named something more standard like "game_name_input" regardless of if its a stub or not. If that's not the case then you can follow the naming convention below. also I would prefer that ending brackets of stub functions are on the line below the definition, but that's just me being picky.

Reply:
Personally, I like to avoid wasting space whenever it is possible to do so without making anything unclear. Stub functions are pretty straightforward by nature.
That said, we're both being picky, lol.